main/getting-started/README.md

```diff
@@ -47,8 +42,8 @@ What you need to do before installing Agoric software and installing the Agoric
   briefly explains the concepts and components of Agoric's Electronic
   Rights Transfer Protocol (ERTP) which is the heart of Agoric
   software.
-4. **[Zoe Introduction](https://agoric.com/documentation/getting-started/zoe-introduction)**: 
-  A Getting Started document that
+4. **Zoe Introduction**: 
+  **[IN PROGRESS]** A Getting Started document that
```

Comment:
Any reason for this not to be a link to the https://agoric.com/documentation/zoe/guide/ in the meantime? Rather than a dead link?